app/services/coronavirus/pages/draft_updater.rb

```diff
@@ -15,33 +15,25 @@ def content_builder
     end
 
     def payload
-      if content_builder.success?
-        Coronavirus::PagePresenter.new(content_builder.data, base_path).payload
-      else
-        raise DraftUpdaterError, content_builder.errors.to_sentence
-      end
+      Coronavirus::PagePresenter.new(content_builder.data, base_path).payload
+    rescue Coronavirus::Pages::ContentBuilder::InvalidContentError,
+           Coronavirus::Pages::ContentBuilder::GitHubConnectionError => e
```

Comment:
Minor, these can be `ContentBuilder::InvalidContentError` since you're in `Cornavirus::Pages`